interactive_engine/executor/ir/core/src/plan/ffi.rs

```diff
@@ -153,6 +153,7 @@ impl From<IrError> for FfiResult {
             IrError::InvalidPattern(s) => FfiResult::new(ResultCode::Others, s),
             IrError::InvalidExtendPattern(err) => FfiResult::new(ResultCode::Others, err.to_string()),
             IrError::PbEncodeError(err) => FfiResult::new(ResultCode::Others, err.to_string()),
+            IrError::PbDecodeError(err) => FfiResult::new(ResultCode::Others, err.to_string()),
```

Comment:
What `Other` error is?

Reply:
Modifed as `ResultCode::ParsePbError` here.